languageserver/server/server.go

```diff
@@ -2349,6 +2369,11 @@ func (s *Server) convertError(
 				err.Name,
 				func(checker *sema.Checker, isFunction bool) insertionPosition {
 					declaration := declarationGetter(checker.Elaboration)
+					if declaration == nil {
```

Comment:
Is returning an empty insertion position the same as not recommending a code action to add the member?

Maybe also add a TODO to look up the getter in the correct checker (it might be imported one)

Reply:
Wilco.
By the way how would we access the correct checker here? we would need to know the location right? but we don't have it. How could I know which checker to get?

Reply:
The interface type / composite type has a location. We need to look up that location's checker, instead using the checker for the current document where the new/undeclared member is used (`maybeAddDeclarationActionsResolver` incorrectly passes the current document's checker to `memberInsertionPosGetter`). But let's do that in a follow-up PR.